documental/filters.py

```diff
@@ -33,7 +33,7 @@ class DocumentalDocsFilter(rest_framework.FilterSet):
     """
     
```

Comment:
Trailing whitespace.

Please use pep8 to format this file.

Reply:
Removed.